From 09ab686d8009503b8a9ccc8947a617d80c038bc8 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Fri, 15 Apr 2016 16:53:47 -0700
Subject: [PATCH] TOOLS-1519 node-sdc-client's make ufds_test target cannot run
 UFDS tests suite

Reviewed by: Pedro P. Candel <pedro@joyent.com>
---
 test/{ufds.tests.js => ufds.test.js} | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename test/{ufds.tests.js => ufds.test.js} (100%)

diff --git a/test/ufds.tests.js b/test/ufds.test.js
similarity index 100%
rename from test/ufds.tests.js
rename to test/ufds.test.js
-- 
2.21.0

